index.js

```diff
@@ -1,37 +1,61 @@
-// A convenience file to allow you to `require` all of the components defined in this module.
+// The main file that is included when you run `require("gpii-express")`.
+//
+// You can optionally load testing support by assigning a variable to the results of `require` and calling
+// `loadTestingSupport`, as in:
+//
+//      var gpiiExpress = require("gpii-express");
+//      gpiiExpress.loadTestingSupport();
+//
+"use strict";
+var fluid = require("infusion");
+var express = fluid.registerNamespace("gpii.express");
+
+var loader = fluid.getLoader(__dirname);
```

Comment:
The "loader" system is an unfortunate API and will be removed - you should go back to the use of raw "require" that you had before. I've been removing this anti-pattern from universal which is presumably from where you copied it